python/lsst/faro/base/MatchedCatalogBase.py

```diff
@@ -252,9 +259,16 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
                 calib_find = (visitPhotoCalibList == visit) & (
                     detectorPhotoCalibList == detector
                 )
-                row = flatPhotoCalibList[calib_find]
-                externalPhotoCalib = row[0].getPhotoCalib()
-                inputs["photoCalibs"][i] = externalPhotoCalib
+                if np.sum(calib_find) < 1:
```

Comment:
It looks like the calibrations can be by visit or by detector, and this logic is intended to account for both possibilities.